composer.json

```diff
@@ -25,24 +25,23 @@
     "require-dev": {
         "ext-intl": "*",
         "doctrine/annotations": "^1.13.3",
-        "laminas/laminas-captcha": "^2.11.0",
```

Comment:
Can you update the `conflict` section as well please?

Reply:
Assuming you want the conflicts to prevent installation of anything less than most recent? Why is this necessary?

Reply:
Only tested versions are supported.
Without the `conflict` section:

1. a user may install an unsupported version
2. find a bug
3. report an Issue here
4. we close it with "Bump your deps"

`conflict` prevents this all

Reply:
Thanks for explaining @Slamdunk 👍